test/e2e/scenario/eventBridge.scenario.ts

```diff
@@ -72,20 +73,25 @@ describe('bridge present', () => {
       })
 
       await flushEvents()
-      expect(intakeRegistry.telemetryErrorEvents.length).toBe(0)
-      expect(bridgeEvents.telemetryErrorEvents.length).toBe(1)
+      expect(intakeRegistry.telemetryErrorEvents.length).toBe(1)
+      expectAllRequestsComeFromBridge(intakeRegistry)
+      intakeRegistry.empty()
     })
 
   createTest('forward logs to the bridge')
     .withLogs()
     .withEventBridge()
-    .run(async ({ intakeRegistry, bridgeEvents }) => {
+    .run(async ({ intakeRegistry }) => {
       await browserExecute(() => {
         window.DD_LOGS!.logger.log('hello')
       })
       await flushEvents()
 
-      expect(intakeRegistry.logsEvents.length).toBe(0)
-      expect(bridgeEvents.logsEvents.length).toBe(1)
+      expect(intakeRegistry.logsEvents.length).toBe(1)
+      expectAllRequestsComeFromBridge(intakeRegistry)
     })
 })
+
+function expectAllRequestsComeFromBridge(intakeRegistry: IntakeRegistry) {
+  expect(intakeRegistry.requests.every((request) => request.isBridge)).toBe(true)
+}
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** similarly to `intakeRegistry.isEmpty()`, ‏what about exposing a method from the intakeRegistry directly?
to have something like:
```ts
expect(intakeRegistry.hasOnlyBridgeRequests()).toBe(true)
```